samples/initialize_checkout_form.py

```diff
@@ -72,7 +72,8 @@
     'buyer': buyer,
     'shippingAddress': address,
     'billingAddress': address,
-    'basketItems': basket_items
+    'basketItems': basket_items,
+    'debitCardAllowed': true
```

Comment:
Az kullanılan bir özellik olduğu için, örnek kodların basitleşmesi adına, sample içine hiç koymamaya ne dersiniz?

Reply:
Olabilir, ben sadece senin php client'ının içinde yaptığın değişikliği kopyaladım. Onun haricinde diğer arkadaşlar yorum yapabilir bu konuya.

Reply:
@onlined'nin de dediği gibi Python'da `True` olarak değer vermek gerekiyor.
referansı şurada -> [https://docs.python.org/2/library/constants.html#True](https://docs.python.org/2/library/constants.html#True)